### DISCUSSION TITLE
How can I uninstall git lfs FROM THE COMPUTER?

### DISCUSSION THREAD

**Discussion by BlackJar72** [git-lfs/git-lfs#4703] [Q&A]
I am NOT trying to remove it from a repository -- the instructions for that were clear and simple.  However, since I'm installed it trying to push to remotes leads to what seems to be an infinite loop and being ask for my user name and password over and over again.  I'm not even looking to just fix that -- I want the lfs software off my computer, completely.

Thank you.

**Comment by bk2204** [comment]
Hey,

Sorry to hear you're having trouble.  Can you tell us what operating system you're using?

**Comment by BlackJar72** [comment]
This was on Linux, Kubuntu 20.10, but the repo was created in Windows.  It turns out now that it only effect the one repo (created in Windows, on NTFS, and probably the only one using lfs).  It seems its not the problem I thought it was -- at the time I was panicking thinking git was broken and I'd not be able use any of my other repos, but they are fine.  I also suspect now the problem has more to do with Linux not working well with NTFS.  Sorry about the up-roar -- at the time I was in a panic thinking all my repos were going to be unusable, but that doesn't seem to be the case.

**Comment by bk2204** [comment]
Hey,

In that case, you can probably remove the `git-lfs` package, either with your preferred graphical interface, or with `sudo apt purge git-lfs`, which should fix the problem if you still want to remove it.  We're also happy to help you figure out the problem and troubleshoot it if you'd like.  I certainly understand how it's frustrating when things don't work and it looks like everything's gone to pot.

**Comment by fommil** [comment]
As this question comes up pretty high for this query thanks to search engine optimisations, it would be great if somebody could also add the instructions for Windows (and potentially OS X) as well. Best that I can see is to remove git with the "add or remove programs" option, then reinstall and untick the LFS box.

**Reply by bk2204** [reply]
That is indeed the proper answer for Windows if you've installed it as part of Git for Windows, which is the usual and recommended way.  As for macOS, it depends on how you installed it.  If you used Homebrew to install it, then you'd use Homebrew to remove it.  Git LFS is not shipped with the Apple Git version, so it will have been installed some other way.

The project does also ship tarballs and zip files with binaries that are relocatable, so if you've installed it via that approach, then you can simply delete the binary from wherever you placed it.
